services/ext/api.go

```diff
@@ -1045,3 +1047,12 @@ func topicsToBloom(topics ...types.TopicType) []byte {
 func TopicsToBloom(topics ...types.TopicType) []byte {
 	return topicsToBloom(topics...)
 }
+
+// GetPasswordStrength uses zxcvbn module and generates a JSON containing information about the quality of the given password
+// (Entropy, CrackTime, CrackTimeDisplay, Score, MatchSequence and CalcTime).
+// userInputs argument can be whatever list of strings like user's personal info or site-specific vocabulary that zxcvbn will
+// make use to determine the result.
+// For more details on usage see https://github.com/status-im/zxcvbn-go
+func (api *PublicAPI) GetPasswordStrength(request *requests.GetPasswordStrength) scoring.MinEntropyMatch {
```

Comment:
Don t we have an auth service or an account part where we could have this function?
@richard-ramos do u know more?

Reply:
Also, worth checking if the node is running when you need to use this endpoint, if it's pre-login it might not be, in which case we generally use c-bindings